girder/api/v1/notification.py

```diff
@@ -112,22 +113,25 @@ def streamGen():
     @autoDescribeRoute(
         Description('List notification events')
         .notes('This endpoint can be used for manual long-polling when '
-               'SSE support is disabled or otherwise unavailable.')
-        .param('since', 'Filter out events before this time stamp.',
-               dataType='integer', required=False)
+               'SSE support is disabled or otherwise unavailable. The events are always '
+               'returned in chronological order.')
+        .param('since', 'Filter out events before this date. Pass as either epoch seconds or '
+               'an ISO-8601 formatted datetime.', required=False)
         .errorResponse()
         .errorResponse('You are not logged in.', 403)
     )
     def listNotifications(self, since):
         user, token = self.getCurrentUser(returnToken=True)
-        timestamp = time.time()
 
         if since is not None:
-            since = datetime.utcfromtimestamp(since)
-        notifications = list(NotificationModel().get(user, since, token=token))
-
-        if notifications:
-            timestamp = max(n['updatedTime'] for n in notifications)
-
-        setResponseHeader('Date', str(timestamp))
-        return notifications
+            try:
+                since = dateutil.parser.parse(since)
```

Comment:
Should we use `girder.utility.parseTimestamp`? It seems to provide a little more robust handling of timezones (if they were included, as ISO-8601 allows).